api/views/workflow.py

```diff
@@ -36,7 +36,7 @@ class ProviderViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = ProviderSerializer
 
     def get_queryset(self):
-        queryset = ShareUser.objects.exclude(robot='').exclude(long_title='')
+        queryset = ShareUser.objects.exclude(long_title='').exclude(favicon='')
```

Comment:
Oh plus we don't want all of the OSF push users to show up (forgot about the main reason).